dashboard/test/controllers/api/v1/teacher_feedbacks_controller_test.rb

```diff
@@ -301,15 +302,19 @@ def parsed_response
 
   # Sign in as teacher and leave feedback for student on level.
   # Assert that the feedback request was successful
-  def teacher_sign_in_and_give_feedback(teacher, student, script, level, script_level, comment, performance)
+  # Note that the section that a piece of teacher feedback is explicitly associated with via section ID
+  # is not (as of Feb. 2021) used in our application (we're only logging section ID for analytics purposes for now),
+  # so the default section created during setup_all is provided unless otherwise overwritten.
+  def teacher_sign_in_and_give_feedback(teacher, student, script, level, script_level, comment, performance, section=@section)
```

Comment:
Would it be cleaner to make `section` a required parameter and pass in `@section` everywhere?

Reply:
Possibly -- I was trying to clarify that the section parameter is different from all of the other parameters, in that it is not varied across tests in order to test some piece of the functionality.

Reply:
Hmmm, shouldn't we test this parameter like the other parameters?

Reply:
It is added in the `can be created` test that it is appropriately saved when the parameter is provided, but it's an optional parameter, which we have several of and don't explicitly test. The only thing I could think of adding would be like "can be created if section ID isn't provided", but maybe seems like overkill since we don't do it for any of the other optional parameters?